autotx/utils/ethereum/helpers/show_address_balances.py

```diff
@@ -1,14 +1,16 @@
 from web3 import Web3
+from gnosis.eth import EthereumNetwork
 
 from autotx.utils.ethereum import get_erc20_balance, get_eth_balance
-from autotx.utils.ethereum.constants import NATIVE_TOKEN_ADDRESS, NetworkInfo
+from autotx.utils.ethereum.constants import NATIVE_TOKEN_ADDRESS
+from autotx.utils.ethereum.networks import SUPPORTED_NETWORKS_CONFIGURATION_MAP, NetworkInfo
 from autotx.utils.ethereum.eth_address import ETHAddress
 
-def show_address_balances(web3: Web3, network: NetworkInfo, address: ETHAddress):
+def show_address_balances(web3: Web3, network: EthereumNetwork, address: ETHAddress):
     eth_balance = get_eth_balance(web3, address)
     print(f"ETH balance: {eth_balance}")
 
-    tokens = network.tokens
+    tokens = SUPPORTED_NETWORKS_CONFIGURATION_MAP.get(network).default_tokens
```

Comment:
Maybe we could add default tokens to network_info?

Reply:
i thought about it but i think its redundant since we have it in the configuration map, also, we're not using it anywhere else

Reply:
Just though it might be simpler to pass in network_info directly instead of network_info.network
I'm good either way